presto-main/src/main/java/com/facebook/presto/operator/project/PageFieldsToInputParametersRewriter.java

```diff
@@ -104,6 +105,12 @@ public RowExpression visitVariableReference(VariableReferenceExpression referenc
         {
             return reference;
         }
+
+        @Override
+        public RowExpression visitDeferredSymbolReference(DeferredSymbolReferenceExpression reference, Void context)
```

Comment:
Yeah. That's a trade-off as we don't want to have commits that do nothing but introduce the unused code.